src/todo/items/todo.ts

```diff
@@ -25,11 +25,36 @@ class Todo extends Item {
     this._lineNextText = val;
   }
 
+  _automaticTodo;
+
+  get automaticTodo () {
```

Comment:
I don't think `automaticTodo` should be a getter/setter. In fact I'm not sure why we need this at all, what difference does it make if a todo has been inserted automatically or not (assuming that's the purpose of the variable)? I think we only care if the related config setting is enabled or not.